src/Http/Response/Factory.php

```diff
@@ -7,8 +7,8 @@
 use Illuminate\Support\Str;
 use Dingo\Api\Http\Response;
 use Illuminate\Support\Collection;
-use Illuminate\Pagination\Paginator;
 use Dingo\Api\Transformer\Factory as TransformerFactory;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
```

Comment:
Yea, I agree it does seem restrictive, but if we use `Illuminate\Contracts\Pagination\Paginator` then fractal won't work at all.

To give some context:
```php
$users = User::paginate(); // returns Illuminate\Pagination\LengthAwarePaginator

// will throw an exception
return $this->response()->pagination($users, new UserTransformer);
```

Or am I missing a trick here?